.travis.yml

```diff
@@ -31,28 +31,21 @@ env:
     - SPRING_OUTPUT_ANSI_ENABLED=ALWAYS
     - SPRING_JPA_SHOW_SQL=false
   matrix:
-    - JHIPSTER=app-ng2-default-from-scratch PROFILE=prod RUN_APP=0
-    - JHIPSTER=app-ng2-cache-nol2 PROTRACTOR=1
-    - JHIPSTER=app-ng2-psql-es-noi18n PROFILE=prod PROTRACTOR=1
-    - JHIPSTER=app-ng2-gradle-jwt-psql PROTRACTOR=1
-    - JHIPSTER=app-ng2-social-websocket PROTRACTOR=1
-    - JHIPSTER=app-ng2-sass-noi18n PROTRACTOR=1
-    - JHIPSTER=app-ng2-gateway-eureka
-    - JHIPSTER=app-ng2-gateway-consul
-    - JHIPSTER=app-ng2-gateway-uaa
-    - JHIPSTER=app-ng2-infinispan
-    - JHIPSTER=app-ng2-cassandra
-    - JHIPSTER=app-ng2-mongodb
-    - JHIPSTER=app-ng2-mongodb-elasticsearch
-    - JHIPSTER=app-ng2-oauth2 PROTRACTOR=1
-    - JHIPSTER=app-react-default
-    - JHIPSTER=app-mysql PROFILE=prod PROTRACTOR=1
-    - JHIPSTER=app-gradle PROTRACTOR=1
-    - JHIPSTER=app-hazelcast-cucumber
-    - JHIPSTER=app-mariadb-kafka
-    - JHIPSTER=app-jwt
-    - JHIPSTER=app-microservice-eureka
-    - JHIPSTER=app-microservice-consul
+    - JHIPSTER=ngx-default PROFILE=prod PROTRACTOR=1
+    - JHIPSTER=ngx-psql-es-noi18n PROFILE=prod PROTRACTOR=1
+    - JHIPSTER=ngx-gradle-fr PROFILE=prod PROTRACTOR=1
+    - JHIPSTER=ngx-mariadb-oauth2-sass-noi18n PROTRACTOR=1
+    - JHIPSTER=ngx-h2mem-social-ws-nol2
+    - JHIPSTER=ngx-mongodb-kafka-cucumber
+    - JHIPSTER=ngx-session-cassandra-fr
+    - JHIPSTER=ngx-couchbase-infinispan
```

Comment:
it could be, but I prefer to use this slot to test 2 options which aren't use a lot couchbase and infinispan. What would you suggest for our main build?
Otherwise, infinispan already has his dedicated build: 
- gradle-ngx-infinispan: https://github.com/hipster-labs/jhipster-travis-build/blob/ngx-gradle/.travis.yml#L71
- for maven: https://github.com/hipster-labs/jhipster-travis-build/blob/ngx-maven/.travis.yml#L57

Same for couchbase

Reply:
I would rather test mostly used options in the main build as not so common options can be in hipster labs and even if they break they arent critical but we would want to know if a PR breaks a common option faster. I would suggest to use infinispan for SQL as it is more common and also an option for cache but no l2 as its easier to break due to all the inter dependencies

Reply:
Good point @deepu105 ! You notice an important thing. It seems the behaviour has changed, as we shouldn't be able to select the cache when it's not SQL.
- In 4.13.0, if it's SQL database, we can choose the cache, otherwise we can't
- In current master, we can choose the cache for noSQL database too (ehcache, hazelcast, infinispan, noL2). Not sure if it works well

I need to review closely this part!